code/modules/world_api/commands/server_query.dm

```diff
@@ -110,7 +110,7 @@
 /datum/topic_command/get_count_mod/run_command(queryparams)
 	var/n = 0
 	for (var/client/client in clients)
-		if (client.holder && (client.holder.rights & R_MOD) && !(client.holder.rights & R_ADMIN))
+		if (client.holder && (client.holder.rights & R_MOD))
```

Comment:
This will probably count HeadAdmin/Dev due to them having all perms

Reply:
That is true, it will account of Head Admin and Head Dev, I assume this is intentional?